### DISCUSSION TITLE
ImportError?

### DISCUSSION THREAD

**Discussion by Eilermoon** [pyinstaller/pyinstaller#6997] [Help] [answered]
Hey all, I've used this in the paste to convert to exe. Trying to run it today, I get this error and no exe in the 'dist' folder. Looks like an issue with my environment? I'm a relative noob so any help would be greatly appreciated!

```ImportError: cannot import name 'Sequence' from 'collections' (C:\Users\*User Name*\AppData\Local\Programs\Python\Python310\lib\collections\__init__.py)```

**Comment by bwoodsend** [comment, marked as answer]
`collections.Sequence` was deprecated and moved to `collections.abc.Sequence`. The cut off version was Python 3.9. Whatever line of code is running `from collections import Sequence` needs replacing with something like `from collections.abc import Sequence`. You'll have to look at the rest of the error message to find out where that is.

**Reply by Eilermoon** [reply]
Thanks! I updated my `pathlib.py` script exactly as you mentioned. Looks like it outputs the exe as desired, but I'm not sure it functions correctly yet. It doesn't execute as expected, the way the .py script does.
I'll run it again and look for further errors.

**Reply by bwoodsend** [reply]
Where is this `pathlib.py`? Is it something you wrote, in a `site-packages` folder or inside Python's `lib` folder? `pathlib` is supposed to be part of the standard library so if it's yours then you should pick a new name for it to stop it clashing with the original or if it's in `site-packages` then you've got the obsolete backport of pathlib and should uninstall it (`pip uninstall pathlib`).

**Reply by Eilermoon** [reply]
Ah yeah it's in ```C:\Users\*User Name*\AppData\Local\Programs\Python\Python310\Lib```.

After trying it again, the .exe it created works correctly. I'll go ahead and uninstall the outdated ```pathlib``` though
